<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="film_cover">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>96</width>
      <height>96</height>
     </rect>
    </property>
    <property name="text">
     <string>cover</string>
    </property>
   </widget>
   <widget class="QLabel" name="film_title">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>title</string>
    </property>
   </widget>
   <widget class="QLabel" name="film_rating">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>rating</string>
    </property>
   </widget>
   <widget class="QLabel" name="film_rank">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>rank</string>
    </property>
   </widget>
   <widget class="QLabel" name="film_country">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>country</string>
    </property>
   </widget>
   <widget class="QLabel" name="film_runtime">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>length</string>
    </property>
   </widget>
   <widget class="QLabel" name="film_director">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>director</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="film_plot">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>371</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>761</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="details_button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>DÃ©tails</string>
    </property>
   </widget>
   <widget class="QPushButton" name="search_history">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>90</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Historique de recherche</string>
    </property>
    <property name="text">
     <string>Historique</string>
    </property>
   </widget>
   <widget class="QListWidget" name="film_actors">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>491</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="opaqueResize">
     <bool>true</bool>
    </property>
    <property name="handleWidth">
     <number>6</number>
    </property>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Titre:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="search_field">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Titre du film</string>
     </property>
    </widget>
    <widget class="QPushButton" name="search_button">
     <property name="toolTip">
      <string>Chercher le film</string>
     </property>
     <property name="text">
      <string>Chercher</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>search_field</tabstop>
  <tabstop>search_button</tabstop>
  <tabstop>search_history</tabstop>
  <tabstop>details_button</tabstop>
  <tabstop>film_plot</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
